### PR TITLE
Azure AD entitlement management: update connected organization description

### PR DESCRIPTION
**Pull request by markwahl-msft**
There are no new APIs or updated resources.  This PR provides more information on identity governance existing connected organization API and resources to clarify how a connected organization identity source of type azureActiveDirectoryTenant comes into existence.

### CODE REVIEW

**Review by FaithOmbongi — APPROVED** [review]
Proposed a few changes.
Q: In the CREATE API, do we also need to add a note for what happens when the caller provides a externalDomainFederation type?

**Review by davidmu1 — APPROVED** [review]

File: api-reference/beta/api/connectedorganization-post.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ The following table shows the properties that are required when you create the [
 |identitySources|[identitySource](../resources/identitysource.md) collection|A collection with one element, the initial identity source in this connected organization.|
 |state|connectedOrganizationState|The state of a connected organization defines whether assignment policies with requestor scope type `AllConfiguredConnectedOrganizationSubjects` are applicable or not. Possible values are: `configured`, `proposed`.|
 
+The single member of the identitySources collection should be of either the [domainIdentitySource](../resources/domainidentitysource.md) or [externalDomainFederation](../resources/externaldomainfederation.md) type.  If the caller provides a domainIdentitySource, the call is successful, and the domain corresponds to a registered domain of an Azure Active Directory tenant, then the resulting connectedOrganization that is created will have an identitySources collection containing a single member of the [azureActiveDirectoryTenant](../resources/azureactivedirectorytenant.md) type.
```
[the remark this answers is not in the document]
Comment (markwahl-msft) [reply api-reference/beta/api/connectedorganization-post.md:57]: I updated the first sentence, but couldn't take your suggestion for the second setencnce, as when this happens, the connectedOrganization remains of the same type, it is only the element in the identitysources collection of the connectedOrganization that changes type.

File: api-reference/beta/resources/azureactivedirectorytenant.md [hunk 2]
```diff
@@ -15,6 +15,8 @@ Namespace: microsoft.graph
 
 Used in the identity sources of an [connectedOrganization](connectedOrganization.md). The `@odata.type` value `#microsoft.graph.azureActiveDirectoryTenant` indicates that this type identifies another Azure Active Directory tenant as an identity source for a connected organization.
 
+When [creating a new connectedOrganization](../api/connectedorganization-post.md), if the caller provides in the identitySources collection a domainIdentitySource, the call is successful, and the domain corresponds to a registered domain of an Azure Active Directory tenant, then the resulting connectedOrganization that is created will have an identitySources collection containing a single member of the [azureActiveDirectoryTenant](azureactivedirectorytenant.md) type.
```
[the remark this answers is not in the document]
Comment (markwahl-msft) [reply api-reference/beta/resources/azureactivedirectorytenant.md]: As with previous comment, I couldn't change to this text, as the connectedOrganization type is not the type that changes, only the identity source inside it.

**Comment by markwahl-msft** [comment]
#sign-off

File: api-reference/beta/resources/azureactivedirectorytenant.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (FaithOmbongi) [reply api-reference/beta/resources/azureactivedirectorytenant.md]: Got it. Resolved.

File: api-reference/beta/api/connectedorganization-post.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (FaithOmbongi) [reply api-reference/beta/api/connectedorganization-post.md:57]: Got it. Resolved.

**Pull request merged**